dask/base.py

```diff
@@ -928,6 +929,9 @@ def normalize_object(o):
     if callable(o):
         return normalize_function(o)
 
+    if dataclasses.is_dataclass(o):
```

Comment:
`is_dataclass` is already imported towards the top of this module. I have a slight preference for using `import dataclasses`, as you've added here, but either way will work. The main thing we want is consistency